data_pipeline/dataset.py

```diff
@@ -34,6 +36,7 @@ def add_eos(training_entry: TrainingEntry, eos_token: str):
 
 def numericalize(training_entry: TrainingEntry,
                  vocabs: Vocabs,
+                 use_shared: bool,
                  glove_embeddings: GloVeEmbeddings):
   """
   Processes the train entry into lists of integeres that can be easily converted
```

Comment:
Please document the new param (use_shared) in the comment. It would be good to also add a doc for glove_embeddings